lib/cocoapods/generator/xcconfig/pod_xcconfig.rb

```diff
@@ -53,6 +53,7 @@ def generate
             'PODS_ROOT' => '${SRCROOT}',
             'PRODUCT_BUNDLE_IDENTIFIER' => 'org.cocoapods.${PRODUCT_NAME:rfc1034identifier}',
             'SKIP_INSTALL' => 'YES',
+            'PODS_TARGET_SRCROOT' => target.sandbox.pod_dir(target.pod_name).to_s
```

Comment:
@benasher44 should this be `to_s` or perhaps always print out the absolute path?